### ISSUE TITLE
Barcode-Scanner

### ISSUE THREAD

**Issue by TVLuke** [Chaotikum/IceBoxWoodBox#4]
An dem Pi in der Icebox hängt per USB ein Barcode-Scanner. Das Kabel ist gefühlt ca. 1 m lang und muss irgendwie aus der Box raus. Der Scanner muss wohl nicht unbedingt an der Box hängen, da die ja eh sehr weit oben steht und der scanner auch nicht grad klein ist.

**Comment by kellertuer** [comment]
Man könnte ja trotzdem wie bei nem alten Telefon an der Seite so ne Gabel machen in der der hängt...

**Comment by TVLuke** [comment]
Ja, aber der Scanner ist ca. so hoch wie die box selbst und für kleine Leute liegt das Ding recht hoch...

Aber: Vielleicht hängt man die Box ja auch seitlich an den Kühlschrank, dann wäre es ne andere Sache.
